go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/stretchr/testify v1.10.0
 	github.com/tidwall/gjson v1.18.0
 	go.uber.org/zap v1.27.0
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
Maybe go ahead with v3 instead?

Reply:
interesting not sure why it auto imported that, change